e2e/pages/events/manageLegalCounsellorsEvent.page.js

```diff
@@ -0,0 +1,45 @@
+const {I} = inject();
+
+module.exports = {
+  fields: function (index) {
+
+    return {
+      legalCounsellor: {
+        firstName: `#legalCounsellors_${index}_firstName`,
+        lastName: `#legalCounsellors_${index}_lastName`,
+        organisationGroup: `#legalCounsellors_${index}_${index}`,
+        email: `#legalCounsellors_${index}_email`,
+        telephone: `[id="legalCounsellors_${index}_telephoneNumber"]`,
```

Comment:
is there a reason you are using this form of locator rather than just the `#legalCounsellors_${index}_telephoneNumber`

Reply:
Not really. Will change it.
Good spot.